projects/core/library/run.py

```
@@ -124,8 +124,8 @@ def run(command, capture_stdout=False, capture_stderr=False, check=True, protect
 
     proc = subprocess.run(command, **args)
 
-    if capture_stdout: proc.stdout = proc.stdout.decode("utf8")
-    if capture_stderr: proc.stderr = proc.stderr.decode("utf8")
+    if capture_stdout and decode_stdout: proc.stdout = proc.stdout.decode("utf8")
```

Flake8 (7.2.0)

[error] multiple statements on one line (colon)

(E701)